Doc/library/dis.rst

```diff
@@ -723,6 +723,47 @@ All of the following opcodes use their arguments.
    are put onto the stack right-to-left.
 
 
+.. opcode:: BUILD_TUPLE_UNPACK (count)
+
+   Pops ``count`` iterables from the stack, joins them in a single tuple,
+   and pushes the result.
```

Comment:
It may be worth to mention that it is used for implementing iterable unpacking in tuple displays (see https://docs.python.org/3/whatsnew/3.5.html#pep-465-a-dedicated-infix-operator-for-matrix-multiplication). Example: `(*x, *y, *z)`.